codegen/test_data/endpoints/bar_bar_method_argnotstruct.gogen

```diff
@@ -54,6 +58,9 @@ func NewBarArgNotStructHandler(deps *module.Dependencies) *BarArgNotStructHandle
 		"bar", "argNotStruct",
 		handler.HandleRequest,
 	)
+	handler.endpointScope = deps.Default.Scope.Tagged(map[string]string{
```

Comment:
let's just reuse this scope: https://github.com/uber/zanzibar/blob/master/runtime/router.go#L70

Reply:
fixed